app/commands.py

```diff
@@ -228,3 +228,23 @@ def run(self, start_date, end_date):
                 process_end_date.isoformat()
             ))
             send_processing_time_for_start_and_end(process_start_date, process_end_date)
+
+
+class PopulateServiceEmailReplyTo(Command):
+
+    def run(self):
+        services_to_update = """
+            INSERT INTO service_email_reply_to(id, service_id, email_address, is_default, created_at)
+            SELECT '{}', id, reply_to_email_address, true, '{}'
+            FROM services
+            WHERE reply_to_email_address IS NOT NULL
+            AND id NOT IN(
+                SELECT service_id
+                FROM service_email_reply_to
+            )
+        """.format(uuid.uuid4(), datetime.utcnow())
```

Comment:
Using uuid.uuid4() here will not work for a insert select statement, because it will use the same uuid for each row.